.env.example

```diff
@@ -9,4 +9,11 @@ APP_ENVIRONMENT=<development|staging|production>
 
 # Optional: Datadog Real User Monitoring
 DATADOG_APPLICATION_ID=<datadog application id>
-DATADOG_CLIENT_TOKEN=<datadog client token>
\ No newline at end of file
+DATADOG_CLIENT_TOKEN=<datadog client token>
+
+# Feature flags
+FEATURE_AB_REDIRECTS=1
```

Comment:
Do we need any way to enable this market-by-market or just global on/off?

Reply:
Global on/off is enough since markets will be addressed in redirect URLs